draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -153,8 +155,7 @@ For any remote host ICE candidate received by the ICE agent, the following proce
 
 4. Otherwise, ignore the candidate.
 
-Multicast DNS resolution might end up retrieving both an IPv4 and IPv6 address.
-In that case, the IPv6 address may be used preferably to the IPv4 address.
+An ICE agent that supports mDNS candidates MUST support the situation where the mDNS resolution resuts in more than one IP address. In this case The ICE agent takes exactly one of the resolved IP addresses and ignores the others. The ICE agent SHOULD use the first IP address resolved.
```

Comment:
What does 'first' mean in the context of v4 vs v6?

Reply:
Clarified